resources/grafana/dashboards/machines.json

```diff
@@ -941,8 +946,8 @@
             "measurement": "total-anomaly-scaled",
             "orderByTime": "ASC",
             "policy": "default",
-            "query": "SELECT mean(/^$Sensor$/) as \"field\" FROM \"original-input\" WHERE (\"machine\" =~ /^$Machine$/) AND $timeFilter GROUP BY time($__interval) fill(null)",
-            "rawQuery": false,
```

Comment:
weird that this ever worked...

Reply:
:man_shrugging: Ran it though the test project and it works for sure now at least.